### ISSUE TITLE
bad fault in ms-dos extender when installing windows 98

### ISSUE THREAD

**Issue by tongpeifu-yh** [86Box/86Box#2949]
**Describe the bug**
Display "bad fault in ms-dos extender" after checking hard disk, with a short GUI flashing across if running slowly.

**To Reproduce**
1. Using configuration like this:
![image](https://user-images.githubusercontent.com/93425406/205891060-69094fc9-9a26-4238-ab2d-0f0b4ed4bfb0.png)

![image](https://user-images.githubusercontent.com/93425406/205891267-6b4502ab-de17-4127-a3a5-53d80457caba.png)

![image](https://user-images.githubusercontent.com/93425406/205891374-b69ea2f8-2fb6-47de-8986-5cbdd6ca7d70.png)

![image](https://user-images.githubusercontent.com/93425406/205891420-506a328e-d1c2-49eb-a25f-c7265d41c132.png)

![image](https://user-images.githubusercontent.com/93425406/205891465-969bcebf-0fa0-41ec-a188-390c8043a143.png)

![image](https://user-images.githubusercontent.com/93425406/205891541-d3898692-acf5-4229-9c18-95ab35a44db7.png)

![image](https://user-images.githubusercontent.com/93425406/205891599-b50aec1e-f0ef-4f0d-b943-98560552988c.png)

2. Go to config the cmos setup,  change the **parallel port mode** to normal
3. Start with windows 98 se boot disk from winworld, run fdisk, restart, format the disk
4. Run setup from win98se cdrom

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/93425406/205892725-9e66b666-61bc-475b-9ac5-2c4f8067297c.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - 86Box version:
![image](https://user-images.githubusercontent.com/93425406/205892893-b2abc874-d005-4276-8634-990fd180ec48.png)

**Additional context**
This problem seems to have appeared once and be repaired after an update. But it appears again.

**Comment by tongpeifu-yh** [comment]
ps: I have tried removing commands about emm386 and himem in config.sys, but it doesn't work at all.

**Comment by TC1995** [comment]
Why 504mb of memory?

**Comment by tongpeifu-yh** [comment]
> 

I just want as large memory as possible.

**Comment by TC1995** [comment]
well, Win98 might show instability when there's a lot of memory.

**Comment by tongpeifu-yh** [comment]
Let me try again. But this memory size works well in another machine type.

**Comment by tongpeifu-yh** [comment]
256MB doesn't work. I'll try 128MB.

**Comment by tongpeifu-yh** [comment]
I choose this machine because I read a recommandation from reddit. And 128MB doesn't work either.

**Comment by OBattler** [comment]
It's the BIOS not being compatible with Winows 9x Setup. Use another machine.
